sql/moz-fx-data-shared-prod/mozilla_vpn_derived/site_metrics_summary_v1/query.sql

```diff
@@ -69,6 +74,22 @@ base_table AS (
       OR (site = "mozilla.org" AND hits.page.pagePath LIKE "%/products/vpn/%")
     )
 ),
+base_table AS (
+  SELECT
+    * EXCEPT (source_param, campaign, content, medium, source),
+    IF(
+      source_param IS NOT NULL,
+      STRUCT(
+        source_param AS campaign,
+        "(not set)" AS content,
+        "(none)" AS medium,
+        CONCAT("www.mozilla.org-", REGEXP_EXTRACT(source_param, "^[^0-9]+")) AS source
```

Comment:
this would be something like `www.mozilla.org-whatsnew`?

Reply:
yes, either `www.mozilla.org-whatsnew` or `www.mozilla.org-welcome`